writer.go

```diff
@@ -208,6 +215,10 @@ func NewWriter(config WriterConfig) *Writer {
 		config.BatchSize = 100
 	}
 
+	if config.MaxBytes == 0 {
+		config.MaxBytes = 1048576
```

Comment:
Please comment on what this number is. Max int32?

Reply:
Oh, I didn't see the comment on the public field at first. Still worth a comment here I think.